en/components/general/update_guide.md

```diff
@@ -41,6 +41,10 @@ Unfortunately not all changes can be automatically updated. Changes bellow are s
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+### From 9.0.x to 10.0.x
+* IgxDropdown
+    * The display property of the dropdown items now is set to display: block. We do this in order to have truncate text enabled by default. Due to that change, if there is more than a simple text in the dropdown item the layout needs to be handled on the application level.
```

Comment:
```suggestion
    * The display property of the dropdown item has been changed from `flex` to `block`. We have done this in order to have truncated text enabled by default. Due to that change, if there is more than text in the content of the dropdown item, the layout needs to be handled on the application level.
```
I would like you to also provide before and after examples of an item that has an icon and text and how to fix the layout.